### DISCUSSION TITLE
Problem with run server

### DISCUSSION THREAD

**Discussion by MaYoRr** [quasarframework/quasar#10641] [Show and Tell]
The server startup does not work(quasar dev). Tried uninstalling and reinstalling the dependencies. On windows it works, on macOS it doesn't. I use the package installer yarn

```
Invalid options object. Stylus Loader has been initialized using an options object that does not match th  e API schema.
   - options has an unknown property 'preferPathResolver'. These properties are valid:
     object { implementation?, stylusOptions?, sourceMap?, webpackImporter?, additionalData? }
  
  - validate.js:105 validate
    [project]/[schema-utils]/dist/validate.js:105:11
  
  - NormalModule.js:547 Object.getOptions
    [project]/[webpack]/lib/NormalModule.js:547:19
```

` "dependencies": {
    "@capacitor/app": "^1.0.2",
    "@capacitor/local-notifications": "^1.0.3",
    "@capacitor/storage": "^1.0.3",
    "@quasar/extras": "^1.10.6",
    "@quasar/icongenie": "^2.4.0",
    "@types/swiper": "^5.4.3",
    "axios": "^0.18.1",
    "core-js": "^3.14.0",
    "luxon": "^2.0.1",
    "quasar": "^2.0.4",
    "signalr-no-jquery": "^0.2.0",
    "swiper": "5.x",
    "vue": "^2.6.14",
    "vue-advanced-cropper": "^1.7.0",
    "vue-awesome-swiper": "^4.1.1",
    "vue-i18n": "^8.25.0",
    "vue-infinite-loading": "^2.4.5",
    "vue-plyr": "^7.0.0",
    "vue-scroll-behavior": "^0.3.0",
    "vue-template-compiler": "^2.6.14",
    "vuex": "^3.6.2"
  },
  "devDependencies": {
    "@capacitor/android": "^3.0.1",
    "@capacitor/browser": "^1.0.1",
    "@capacitor/camera": "^1.0.1",
    "@capacitor/cli": "^3.0.1",
    "@capacitor/core": "^3.0.1",
    "@capacitor/device": "^1.0.0",
    "@capacitor/filesystem": "^1.0.1",
    "@capacitor/ios": "^3.0.0",
    "@capacitor/network": "^1.0.1",
    "@capacitor/push-notifications": "^1.0.0",
    "@capacitor/splash-screen": "^1.0.0",
    "@quasar/app": "^3.1.0",
    "babel-eslint": "^10.0.1",
    "cordova-plugin-file-opener2": "^3.0.5",
    "eslint": "^6.8.0",
    "eslint-config-prettier": "^6.9.0",
    "eslint-loader": "^3.0.3",
    "eslint-plugin-vue": "^6.1.2",
    "husky": "^6.0.0",
    "prettier": "^1.18.2",
    "pretty-quick": "^3.1.0"
  },`

**Comment by ontwikkelfabriek** [comment]
@MaYoRr 


- please post the output of `quasar info` on from machines ( and compare see any difference,  nodejs version, quasar versions, yarn all)
- remove node_modules and run 'yarn' again

> I use the package installer yarn

- use **npm** for global libs  ( like quasar cli)
- use **yarn** for project  ( project deps in package.js)
